superset/utils/log.py

```diff
@@ -133,8 +155,29 @@ def log_context(  # pylint: disable=too-many-locals
             records=records,
             dashboard_id=dashboard_id,
             slice_id=slice_id,
-            duration_ms=round((time.time() - start_time) * 1000),
+            duration_ms=duration,
```

Comment:
We need to convert to int:

```suggestion
            duration_ms=duration.total_seconds() * 1000,
```